### PR TITLE
Small tweaks to various default cards

### PR DESCRIPTION
**Pull request by 13robin37**
- Optimized the code for some cards, adjusted documentation.
- Added a tap variable for the light popup to open it with a single tap instead of holding the light icon.

Someone might want to verify the code for the media player card and alarm chip, I don't use those actively. Also found out that the chip_back works when supplying a default path, maybe someone could test that with his/her installation as well - worked for both my dev and prod environment. Adjusted the docu to reflect the change.

### CODE REVIEW

File: custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml [hunk 1]
```diff
@@ -135,27 +125,18 @@ card_media_player:
...
+                "google podcasts": "mdi:google-podcast",
+                "plex": "mdi:plex",
+                "soundcloud": "mdi:soundcloud",
+                "youtube music": "mdi:youtube",
+                "oto music": "mdi:music-circle",
+                "pandora": "mdi:pandora",
+                "netflix": "mdi:netflix",
+                "bluetooth audio": "mdi:bluetooth"
               }
-            }
```
Comment (basbruss) [inline custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml:157]: This line is needed to close off the if/else statement

**Review by basbruss — APPROVED** [review]

**Comment by basbruss** [comment]
I also noticed the list on the developing page of the wiki is less readable:
Screen Shot 2022-06-01 at 14 04 03

**Review by basbruss — APPROVED** [review]
Great work!

**Comment by 13robin37** [comment]
Closes #643

**Pull request merged**